doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -0,0 +1,64 @@
+
+## Setup
+
+First, you need to run the Node-RED by using the following command in terminal:
+```bash
+$ node-red 
+```
+```bash
+# Tip: Node-RED runs locally on port 1880 by default, so make sure to allow the service to run on this port with:
+
+$ sudo ufw allow 1880
+```
+
+>Note: you have to [secure Node-RED](https://nodered.org/docs/user-guide/runtime/securing-node-red) if you want to connect it to MQTT Server over TLS, since its editor is not secured by the default.
+
+If Node-RED is sucessfully launched, when navigating to `https://localhost:1880` in your web browser you should see something like:
+
+{{< figure src="nodered_dashboard.png" alt="Node-RED dashboard" >}}
+
+On the left side, you can see various types of nodes (they can also be found in the [Node-RED library](https://flows.nodered.org/)) that can be used in order to build flows, although in this guide only a few simple inbuilt nodes are used. 
+
+The Console provides the information needed for completing this integration, under **MQTT** submenu from **Integrations** menu within desired application.
+
+{{< figure src="console_info.png" alt="MQTT Server connection information" >}}
+
+In this example, since TTS is running locally, the inbuilt MQTT Server is deployed on port 1883 for insecure connection and on port 8883 for the TLS-secured one.
+
+## Receive events and messages
+
+This section follows the process of setting up a flow which subscribes and listens to the events and messages that are being published by the MQTT Server. 
+
+First, find the `mqtt in` node in the library section and place it on the flow dashboard. This node connects to the MQTT broker and subscribes to messages from the specified topic. Double-click on the node to configure its properties.
+
+{{< figure src="node_properties.png" alt="mqtt in node properties" >}}
+
+In the **Server** dropdown menu, select **Add new mqtt-broker** and click on the button besides to edit it. 
+
+{{< figure src="node_connection.png" alt="COnfiguring MQTT Server connection information" >}}
+
+In the **Connection** tab, under **Server**, you should provide MQTT server's URL - if it runs locally, then type **localhost**. The **Port** value depends on whether the connection is to be set up as a secure one or not. In this example, we are using TLS, so the **Port** value is set to 8883. In this case you should also check the **Enable secure (SSL/TLS) connection** box.
+
+{{< figure src="node_security.png" alt="Configuring MQTT Server credentials" >}}
+
+In the **Security** tab, enter the **Username** and **Password** according to values in the Console.
+
+Under **Topic**, a topic to listen to can be specified, but for the testing purposes set the value to `#` (all topics). The **QoS** value can be arbitrarily chosen, as well as the **Output**. In this example, we want to see each message as `a parsed JSON object`. 
```

Comment:
1. Shouldn't we recommend subscribing to specific topics instead of using "all"?
2. QoS is not arbitrary. Let's leave it at 0.

Reply:
1. Subscribing to specific topics is already explained a step back in the docs, in https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/doc/content/integrations/mqtt/_index.md, and there is also a usage of '#' for topic name (I guess to make it easier to follow). Do you still think that I should change this?
2. I guess I meant arbitrarily chosen from the listed options according to one's needs, but I will rephrase this as you said.